wiki/docs/developer-resources/functional-spec.md

```diff
@@ -0,0 +1,218 @@
+
+![Accessing settings](functional-spec-images/accessing-settings.png)
+
+
+If a passcode is set up it needs to be entered correctly before the setting page can be accessed:
+
+![Entering passcode](functional-spec-images/enter-passcode.png)
+
+The settings page contains tabs for general settings and view-specific settings. Each settings tab allows the user to make settings that are inherited by all satellites as well as to override settings per satellite. This gives the user maximum flexibility. All settings tabs use the same 'matrix' structure to visualize these settings.  By default the general settings tab is selected:
+
+![General settings](functional-spec-images/general-settings.png)
+
+By default the "global" setting value are inherited by all sattelites, but can be overriden per satellite by selecting the 'override' checkbox in the correct cell. That enables the UI element to change the default value, as the image above shows for the passcode setting for Satellite B. In the image above, all other global settings are inherited to both Sattelite A and B.
+
+Any setting changes made are applied immediately. The setting page can be closed using the same button as was used to open it. This button is available on all settings pages.
+
+The following settings pages and buttons are available:
+- general settings
+- view-specific settings (clock, weather, etc)
+- add view button
+- access control settings
+
+#### General settings
+The general settings tab provides settings that are not view specific. The following settings are provided. This is not an exhaustive list and will be expanded in the future. Also, this list is not necessary going to be shown in this order:
+
+|Setting name|Required yes/no|Type|Default value|Notes|
+|---|---|---|---|---|
+|**Passcode**|No|Whole Number|None|Passcode to access the settings page|
+|**Default view**|Yes|View|Clock|The home screen to show on the satellite|
+|**Timeout (s)**|Yes|20|Whole Number|Timeout in seconds after which the satellites switch back to its default view|
+|**Default mode**|Yes|Enum (normal, night)|Normal|Available options: normal, night|
+|**Show top bar**|Yes|Boolean|False|Show or hide bar at top of the screen to provide touch access to views, similar to top bar in HA dashboards that allows access to views|
+|**Default do not disturb state**|Yes|Boolean|False||
+|**Launch icons**|No|String|Empty string|Need to flesh this out!|
+|**Status icon size**|No|Whole number|?|Need to flesh this out!|
+|**Font style**|Yes|String|Roboto|Options: Roboto, neeed to provide dropdown of options|
+|**24-hour clock**|Yes|Boolean|Dependent on HA config||
+|**Assistant**|Yes|Assistant entity name|Default HA assistant||
+|**TTS**|Yes|HA entity name in `tts` domain|If only one TTS entity select that as default|The TTS entity to use for any system generated messages|
```

Comment:
This is for TTS outside of Assist associated TTS?

Reply:
I would suggest so yes - I had the same initial thought too, but realised things like broadcast isn't possible without defining the TTS entity.

Perhaps something we should investigate as an enhancement to core along side the `assist_pipeline`